apps/internal/base/base.go

```diff
@@ -257,6 +268,72 @@ func (b Client) AcquireTokenByAuthCode(ctx context.Context, authCodeParams Acqui
 	return b.AuthResultFromToken(ctx, authParams, token, true)
 }
 
+// AcquireTokenByInteractive initiates the interactive auth flow.
```

Comment:
@abhidnya13 @keegan-caruso is there canonical documentation for interactive login that I could paste a link to?  I did a little searching but couldn't find anything.

Reply:
unless @abhidnya13 has something better

https://docs.microsoft.com/en-us/azure/active-directory/develop/msal-authentication-flows#interactive-and-non-interactive-authentication

Reply:
https://docs.microsoft.com/en-us/azure/active-directory/develop/scenario-desktop-acquire-token?tabs=dotnet#acquire-a-token-interactively 

I think we should have a wiki page that looks something like this to help customers use our stuff easily. https://github.com/AzureAD/microsoft-authentication-library-for-java/wiki/Acquiring-Token-Interactively
Can track it an a different issue.

Reply:
@abhidnya13 since your link doesn't cover Go (for obvious reasons) I'm thinking we should use the link provided by @keegan-caruso for now.  Once we have a better link we can update the docs.  Does that work?

Reply:
Sounds good !